docs/custom_model.md

```diff
@@ -17,24 +17,36 @@ Follow the [Getting Started](getting_started.md) guide to set up the infrastruct
 
 Take a look at the [Repo Details](code_description.md) document for a description of the structure of this repository.
 
-## Follow the Bootstrap instructions
+## Bootstrap the project
 
-The [Bootstrap from MLOpsPython repository](../bootstrap/README.md) guide will help you to quickly prepare the repository for your project.
+Bootstrapping will prepare the directory structure to be used for your project name which includes:
+
+* renaming files and folders from the base project name `diabetes_regression` to your project name
+* fixing imports and absolute path based on your project name
+* deleting and cleaning up some directories
 
 **Note:** Since the bootstrap script will rename the `diabetes_regression` folder to the project name of your choice, we'll refer to your project as `[project name]` when paths are involved.
 
+To bootstrap from the existing MLOpsPython repository:
+
+1. Ensure Python 3 is installed locally
+1. From a local copy of the code, run the `bootstrap.py` script in the `bootstrap` folder
+`python bootstrap.py -d [dirpath] -n [projectname]`
+    * `[dirpath]` is the absolute path to the root of the directory where MLOpsPython is cloned
+    * `[projectname]` is the name of your ML project
+
 ## Configure training data
 
 The training ML pipeline uses a [sample diabetes dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_diabetes.html) as training data.
 
-To use your own data:
+**Important** Convert the template to use your own model Azure ML Dataset for model training via these steps:
```

Comment:
I think the first instance of 'model' isn't necessary =>
Convert the template to use your own Azure ML Dataset for model training via these steps